addons/analytic/models/analytic_plan.py

```diff
@@ -145,17 +149,30 @@ def _compute_analytic_account_count(self):
         for plan in self:
             plan.account_count = len(plan.account_ids)
 
+    def _compute_all_children_ids(self):
+        self.env.cr.execute("""
+            SELECT parent.id,
+                 array_agg(child.id) as children_ids
+            FROM account_analytic_plan parent
+            JOIN account_analytic_plan child ON child.parent_path LIKE parent.parent_path || %s
```

Comment:
No need to interpolate for this
```suggestion
            JOIN account_analytic_plan child ON child.parent_path LIKE parent.parent_path || '%%'
```